variables.tf

```diff
@@ -36,6 +36,26 @@ variable "dependency_ids" {
   default = {}
 }
 
+variable "target_revision" {
+  description = "Override of target revision of the application chart."
+  type        = string
+  default     = "v1.0.0" # x-release-please-version
+}
+
```

Comment:
I find it strange you needed to add this variable as I already did this when adding the release-pipeline, otherwise it would not work. I could have forgotten to add the variable to the variants, but not in `main.tf` and `variables.tf`. Are you sure your branch is updated with the main branch?

Maybe this is the reason why you also pointed to `main` on your PR in the cert-manager module...

Reply:
I removed the duplicated attribut